features/unsupported/USBDevice/USBAudio/USBAudio.cpp

```diff
@@ -113,9 +113,12 @@ bool USBAudio::write(uint8_t * buf) {
     return true;
 }
 
-void USBAudio::writeSync(uint8_t *buf)
+void USBAudio::writeSync(uint8_t *buf, int jitter_nb)
 {
-    USBDevice::writeNB(EPISO_IN, buf, PACKET_SIZE_ISO_OUT, PACKET_SIZE_ISO_OUT);
+    if ((jitter_nb != -1) && (jitter_nb !=1)) {
```

Comment:
What's the valid range for ``jitter_nb`` ? what is this condition handling? It's signed integer, but the documentation says number of samples, thus I would imagine this to be unsigned <0, int_max> ?